carservice/views.py

```diff
@@ -83,6 +84,51 @@ def post(self, request):
             return redirect('/car/read/')
         return render(request, 'car_delete.html', {'form': form})
 
+@login_required
+def carsearch(request):
+    search = request.GET.get('search')
+    cars = Car.objects.none()
+    offers = Offer.objects.none()
+    message = ''
+
+    if search:
+        search_terms = search.split()
+        car_brand = ''
+        car_model = ''
+
+        if len(search_terms) > 0:
+            car_brand = search_terms[0]
+            if len(search_terms) > 1:
+                car_model = search_terms[1]
```

Comment:
a co jeśli się zamieni kolejność, albo tylko wpisze model? wrzuć jakiegoś screena do pr jak wygląda ta szukajka to trochę łatwiej będzie ocenić

Reply:
https://gyazo.com/a6260f05a562e72ace463b091cf166a2 <- szukanie marki
https://gyazo.com/8d5ec8112db7ff5c8506d2f13925f34c <- szukanie modelu
https://gyazo.com/40b628be228a77d3c8307c10b7f2f8c5 <- marka + model

https://gyazo.com/0351094f674257820812e6f3c43fa645 <- po kliknięciu na ofertę